tsptw_data_dt.h

```diff
@@ -959,6 +1006,16 @@ void TSPTWDataDT::LoadInstance(const std::string& filename) {
     v->max_ride_distance_ = vehicle.max_ride_distance();
     vehicles_day_.push_back(vehicle.day_index());
 
+    // Estimate vehicle necessary horizon due to time matrix
+    std::vector<int64> max_times = v->MaxTimes(size_matrix_);
```

Comment:
Most vehicles use the same matrix, we will be parsing the time matrix for no good reason.